src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -135,6 +148,53 @@
                   </button>
                 </span>
               </form>
+              <form
+                #editForumUrl="ngForm"
+                class="form-inline"
+                fxLayout
+                *ngIf="!(isNFL$ | async) && workflow?.mode !== WorkflowType.ModeEnum.DOCKSTOREYML"
+              >
+                <div class="form-group" fxFlex="noshrink" fxLayout fxLayoutAlign=" center">
+                  <strong matTooltip="Link to a discussion forum for this workflow">Discussion Forum</strong>:
+                  <a *ngIf="!forumUrlEditing" [href]="workflow?.forumUrl" fxFlexOffset="4px"> {{ workflow?.forumUrl }} </a>
+                  <input
+                    *ngIf="forumUrlEditing"
+                    minlength="3"
+                    maxlength="256"
+                    type="text"
+                    class="input-default form-control"
+                    name="forumUrl"
+                    [(ngModel)]="workflow.forumUrl"
+                    placeholder="Link to discussion forum"
+                    fxFlex="noshrink"
+                    fxFlexOffset="4px"
```

Comment:
Ideally no pixels, but the pattern already exists, so I guess you'd have to change it everywhere on the form. :( Don't bother, unless Gary pushes for it.